utils/utils.go

```diff
@@ -48,6 +48,9 @@ const (
 
 	windowsOsType = "windows"
 	homeDirPrefix = "~/"
+
+	// DefaultChannelAddress is the default local network address that user application listen on.
+	DefaultChannelAddress = "127.0.0.1"
```

Comment:
Shall we call it this ?
```suggestion
	DefaultAppChannelAddress = "127.0.0.1"
```

Reply:
done